utilities/blob_db/blob_db_test.cc

```diff
@@ -25,7 +25,22 @@ class BlobDBTest : public testing::Test {
  public:
   const int kMaxBlobSize = 1 << 14;
 
-  BlobDBTest() : dbname_(test::TmpDir() + "/blob_db_test"), blob_db_(nullptr) {
+  class MockEnv : public EnvWrapper {
```

Comment:
db_test_util.h already has a `SpecialEnv` with facilities to mock time. Why not use them instead of creating your own? 
Granted I haven't used them myself, and am just about to investigate use them for FIFOWithTTL tests ... so I am not sure if they work for you. But seems like multiple others tests are making use of those functions successfully.